core-scroll-header-panel.css

```diff
@@ -26,6 +26,12 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   overflow-y: auto;
 }
 
+/* github.com/Polymer/core-scroll-header-panel/pull/38*/
+#mainContainer {
```

Comment:
There's a #mainContainer selector right above this one. Can they be collapsed into one?